### PR TITLE
finished about us section + computerwindowcomponent fixes

### PR DESCRIPTION
**Pull request by nyssaaftab**

### CODE REVIEW

File: src/styles/components/ComputerWindow.module.css [hunk 1]
```diff
@@ -1,10 +1,13 @@
 .topbarButtons {
   border-radius: 100%;
   height: 0.5rem;
-  list-style: none;
-  position: absolute;
+  /* list-style: none;
```
Comment (inolasv) [inline src/styles/components/ComputerWindow.module.css:4]: you can comments like this if they're not doing anything

File: src/styles/sections/AboutUsSection.module.css [hunk 2]
```diff
@@ -1,29 +1,98 @@
...
+  background-color: var(--light-light-pink);
+  width: 60%;
+}
+
+.rightPanel {
+  padding: 1.5rem;
+}
+
+.diversityContainer {
+  /* not sure why this isn't working to extend the width of the statement */
```
Comment (inolasv) [inline src/styles/sections/AboutUsSection.module.css:74]: if its fixed, can remove this comment

**Review by inolasv — CHANGES_REQUESTED** [review]
Looks good for the most part, the section does seem a bit off centered (or left aligned right now), if you could fix that. Also, you can remove the old about us section.

image

Also, can you fill out the pr description next time? Just like a summary of changes and screenshots are the main things we'd want in there.

**Comment by nyssaaftab** [comment]
fixed alignment and removed old about us section

File: src/pages/index.js [hunk 3]
```diff
@@ -105,7 +105,7 @@ export default function Home() {
         )}
         {/* */}
 
-        <h1 className={styles.title}>Illinois Women in Computer Science</h1>
+        {/* <h1 className={styles.title}>Illinois Women in Computer Science</h1>
```
Comment (inolasv) [inline src/pages/index.js:108]: could you just remove this instead of commenting it out

**Review by inolasv — APPROVED** [review]

**Pull request merged**